src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: Tutorial 1 Hello World
+---
+
+# Tutorial 1: Hello World
+
+This process ties into the Hello World sample application available [here](https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld).
+The important concepts to get out of this tutorial are:
+Orleans is not a single, standalone application that you download and install.
```

Comment:
this line indicates that we are assuming people might think Orleans is a single, standalone application. But I haven't seen people confuse Orleans with a single standalone application yet. So personally I don't think we should start with this, I'm afraid this may offend people.